src/kernel/src/fs/null/vnode.rs

```diff
@@ -33,6 +32,36 @@ fn access(vn: &Arc<Vnode>, _td: Option<&VThread>, access: Access) -> Result<(),
     todo!();
 }
 
+fn getattr(vn: &Arc<Vnode>) -> Result<VnodeAttrs, Box<dyn Errno>> {
+    todo!()
+}
+
+fn lookup(vn: &Arc<Vnode>, td: Option<&VThread>, name: &str) -> Result<Arc<Vnode>, Box<dyn Errno>> {
+    let null_mount: &NullNode = vn.data().downcast_ref().unwrap();
+
+    let lower = null_mount
```

Comment:
There is a PS4 specific code before this one so better to put a `TODO` here.

Reply:
I already asked about it I think in one of your PR's we don't support cn_nameiop. That code only has to do with that if I'm not mistaken.

Reply:
Sorry I mistaken this one is `open`.